.dockerignore

```
@@ -45,3 +45,7 @@ venv/
 **/videos/
 **/screenshots/
 .DS_Store
+
+# Dev files
+.pre-commit-config.yaml
+.fides/fides.toml
```

we should exclude this whole directory, as it also includes the fides files

```suggestion
.fides/
```